content/articles/premium-domains.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Premium Domains
+excerpt: In the new gTLDs domain space, a premium domain is a domain that has been considered of special value by a registrar.
+categories:
+- Domains
+---
+
+# Premium Domains
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## What is a Premium Domain?
+
+A premium domain is a domain that has been considered of special value by a registrar. Premium domains generally are available in the new gTLDs domain space. They are usually more expensive that regular domains, prices going up to several thousand dollars for one domain in some cases.
```

Comment:
Would suggest: "A premium domain is a domain that is priced differently by a registrar and generally found in the new gTLDs space. They are typically more expensive than a normal domain with prices going into the thousands of dollars in some cases. An example of a gTLD is `.love` or `.ninja`"